package.json

```diff
@@ -32,6 +34,7 @@
     "@testing-library/vue": "^5.8.2",
     "@vueuse/core": "^5.2.0",
     "core-js": "^3.16.0",
+    "dotenv": "^10.0.0",
```

Comment:
I _strongly_ urge you to use [`dotenv-safe`](https://www.npmjs.com/package/dotenv-safe) (and with that, `@types/dotenv-safe`) instead: it uses `dotenv` under the hood, but with the addition that it throws if `.env` and `.env.example` don't have the exact same fields contained in them. This prevents any env-related surprises down the line. It _does_ use an older version of `dotenv` however (8.2.0, as opposed to 10.0.0), but I don't expect that to be a dealbreaker, looking at the [changelog](https://github.com/motdotla/dotenv/blob/master/CHANGELOG.md).